.mvn/wrapper/settings.xml

```diff
@@ -255,6 +255,15 @@ under the License.
           <url>https://repo1.maven.org/maven2</url>
         </repository>
       </repositories>
+      <pluginRepositories>
```

Comment:
@ochaloup if this was not something we were doing before and relates to downloads over HTTP vs HTTPS then I think we should have it in a separate commit

Reply:
@tomjenkinson ok. do I need a separate JIRA for this or is it ok to just mention it in the commit description?